ehitchhiking-ui/src/app/shared/components/blacklist/blacklist.component.ts

```diff
@@ -1,4 +1,4 @@
-import {Component, OnInit} from '@angular/core';
+import {Component, Input, OnInit} from '@angular/core';
```

Comment:
The import of Input is not required here. You don't use Input in the your code

Reply:
deleted